pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -131,10 +138,9 @@ type ServingRuntimeStatus struct {
 
 // ServerType constant for specifying the runtime name
 // +k8s:openapi-gen=true
-// +kubebuilder:validation:Enum=triton;mlserver
 type ServerType string
 
-// ServerType Enum
+// Built-in ServerTypes (others may be supported)
 const (
 	// Model server is Triton
 	Triton ServerType = "triton"
```

Comment:
If there are values used elsewhere, should they be added to this list?
In particular, I'm thinking about the addition of the `"ovms"` server type that would be added in https://github.com/kserve/modelmesh-serving/pull/141.

Reply:
The list isn't meant to necessarily be exhaustive. I wasn't sure about adding `ovms` to the interface since it's still kind of beta, but I suppose this won't show up in the actual CRD so no harm to add it here.